src/nvim/api/private/helpers.c

```diff
@@ -2078,3 +2081,17 @@ bool parse_float_config(Dictionary config, FloatConfig *fconfig, bool reconf,
   }
   return true;
 }
+
+/// Helper function tos et current_sctx for api calls
+///
+/// @param channel_id     api clients id. Used to determine if it's a internal
+///                      call or a rpc call.
+void set_api_sctx(uint64_t channel_id)
```

Comment:
use consistent prefixes. other API functions (should) have `api_` as the prefix.

```suggestion
void api_set_sctx(uint64_t channel_id)
```

Reply:
I'm not sure what you meant here. This isn't an API function rather a helper like `modify_keymap()`

Reply:
but it's related to api subsystem. Seems useful to have such functions with a common prefix instead of random prefixes.

Reply:
Ok . I'll change it .

Reply:
Done